pkg/indexmanagement/validations_mappings_test.go

```diff
@@ -25,7 +25,7 @@ var _ = Describe("Index Management", func() {
 			Spec: esapi.ElasticsearchSpec{
 				IndexManagement: &esapi.IndexManagementSpec{
 					Policies: []esapi.IndexManagementPolicySpec{
-						esapi.IndexManagementPolicySpec{
+						{
```

Comment:
neat! what was updated that lets us infer this now?

Reply:
IIRC, gofmt is responsible for this. Making `fmt` a dependency target for the `build` target yields this output.